KsApi/models/ProjectStatsEnvelopeTests.swift

```diff
@@ -23,6 +23,11 @@ final class ProjectStatsEnvelopeTests: XCTestCase {
       ["date": 555444335]
     ]
     let json: [String: Any] = [
+      "referral_aggregates": [
+        "custom": 1.0,
+        "external": "15.0",
```

Comment:
Just curious, how come `external` is the only string?

Reply:
we get a string value from the server. there is a helper function in the `ProjectStatsEnvelope` called `stringToDouble` that we use to convert it into a `Double`

Reply:
Yeah I just meant its strange that the API returns it in this way. Ideally they would be consistent, but that's not part of this ticket 👍